source/arm9/console.c

```diff
@@ -414,23 +411,37 @@ void consoleLoadFont(PrintConsole* console) {
 		} else {
 			console->fontCurPal = 15 << 12;
 
-			for (i = 0; i < console->font.numChars * 16; i++) {
-				u16 temp = 0;
-
-				if(console->font.gfx[i] & 0xF)
-					temp |= 0xF;
-				if(console->font.gfx[i] & 0xF0)
-					temp |= 0xF0;
-				if(console->font.gfx[i] & 0xF00)
-					temp |= 0xF00;
-				if(console->font.gfx[i] & 0xF000)
-					temp |= 0xF000;
-
-				console->fontBgGfx[i] = temp;
+			if (console->font.bpp == 1) {
+				for (i = 0; i < console->font.numChars * 8; i++) {
+					u8 row = ((const u8*) console->font.gfx)[i];
+					u32 temp = 0;
+					if(row & 0x80) temp |= 0xF;
```

Comment:
The order may appear reversed here. This is to match the typical bit order of PC/DOS character sets, as well as the 1bpp tile editing tools I've encountered such as YY-CHR.